spyder/plugins/switcher/plugin.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2009- Spyder Project Contributors
+#
+# Distributed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Switcher Plugin.
+"""
+
+# Third-party imports
+
+# Local imports
+from spyder.api.translations import _
+from spyder.api.plugins import Plugins, SpyderPluginV2
+from spyder.api.plugin_registration.decorators import on_plugin_available, on_plugin_teardown
+from spyder.plugins.switcher.confpage import SwitcherConfigPage
+from spyder.plugins.switcher.container import SwitcherContainer
+from spyder.plugins.mainmenu.api import (ApplicationMenus, FileMenuSections)
+
+
+# --- Constants
+# ----------------------------------------------------------------------------
+class SwitcherActions:
+    FileSwitcherAction = 'file switcher'
+    SymbolFinderAction = 'symbol finder'
+
+
+# --- Plugin
+# ----------------------------------------------------------------------------
+class Switcher(SpyderPluginV2):
+    """
+    Switcher plugin.
+    """
+
+    NAME = "switcher"
+    REQUIRES = []
```

Comment:
```suggestion
```

This class constant is empty by default.

Reply:
Okay! Sorry a lot of this file was generated by the cookie cutter template so I thought it was necessary